src/operator_repo/core.py

```diff
@@ -204,9 +204,9 @@ def __eq__(self, other: Any) -> bool:
         if self.csv_operator_name != other.csv_operator_name:
             return False
         try:
-            return Version.parse(
+            return Version(  # type: ignore
```

Comment:
Unfortunatelly, semantic_versioning is untyped, therefore mypy interprets the `__eq__` and others as returning `Any`. Also in the new library, the `parse` method returns a tuple (which is also comparable), but I figured storing it in an object might be beneficial in the future? I am open to feedback on this.